Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -254,5 +267,59 @@ export default class LinearTimelineComponent extends Vue {
             container[0].focus();
         });
     }
+
+    private goToPageByCurrentDate(currentDate: Date) {
```

Comment:
The logic for this already exists on the method ```setPageFromDate```. How about spliting that after the nextTick section and re-using it?